### ISSUE TITLE
pod install error

### ISSUE THREAD

**Issue by deter3** [BenjaminPrieur/IOStickyHeader#4]
run pod install in example folder ,

[!] Unable to satisfy the following requirements:
- `IOStickyHeader (from`../`)` required by `Podfile`
- `IOStickyHeader (from`../`)` required by `Podfile`
- `IOStickyHeader (= 0.1.0)` required by `Podfile.lock`

xcode 7.0.1 swift 2.0 

can you fix it ? 

P.S. btw , thanks for pointing out need to implement the custom flow layout , you saved me lots of time .

**Comment by BenjaminPrieur** [comment]
Did you keep the example project in the IOStickyHeader directory?

Other solution is to replace  " pod "IOStickyHeader", :path => "../" " with " pod "IOStickyHeader" " in Podfile

keep me posted if it fixed your problem

**Comment by deter3** [comment]
1. I kept example folder .
2. I tried " pod "IOStickyHeader" ,  and pod install in example folder ,  installed successfully . when build project , the result is 
   "diff: /../Podfile.lock: No such file or directory
   diff: /Manifest.lock: No such file or directory
   error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation."
3. my cocoapods is 0.39 , xcode 7 .0.1 .
